dist/api_data.py

```diff
@@ -847,9 +847,10 @@ def __cmp__(self, other):
 
 'WT_SESSION.drop' : Method([
     Config('checkpoint_wait', 'true', r'''
-        wait for the checkpoint lock, if \c checkpoint_wait=false, perform
-        the drop operation without taking a lock, returning EBUSY if the
-        operation conflicts with a running checkpoint''',
+        wait for concurrent checkpoints to complete before attempting the drop
+        operation. If \c checkpoint_wait=false, attempt the drop operation
+        without waiting, returning EBUSY if the operation conflicts with a
+        running checkpoint''',
```

Comment:
trying to clarify that `checkpoint_wait` isn't about locking, that is, `lock_wait` configures whether or not we "wait for the checkpoint lock", `checkpoint_wait` only configures if we try to acquire the checkpoint lock.